02-about.Rmd

```diff
@@ -0,0 +1,80 @@
+* multiple levels of isolation between users and system components
+  
+Resilience and high availability means analysts can depend on AP to store their work and data, and to complete time–sensitive and business–critical projects.
+
+## What it isn't for
+
+AP does a lot, but data is a big subject and AP doesn't try to do everything. These things are currently outside scope:
+
+### Production apps at scale
+
+Apps on AP should be restricted to private prototypes or for small numbers of staff users (e.g. less than 50). A key benefit of AP is that analysts and data scientists have all the tools to be creative and experimental with interactive dashboards and apps, in a safe sandbox environment. In addition, simple apps for a few staff e.g. for searching through data are fine. However the AP team aims to rapidly change and improve the apps deployment and hosting, so sets an apps availability expectation of only 95% at this stage. Once an app is proven as an AP prototype then it should be adopted by a MoJ Digital & Technology service team to be developed by a multi-disciplinary team and meet the Service Standard.
+
+### MI (Management Information) systems
+
+MI is data collected and used by a service team to track and manage their service day-to-day. In contrast AP has been designed for analysts who produce official statistics and generally work on longer term decision-making with policy teams. There is a view that the user needs differ between these two groups - a service team's business/performance analyst might look at standard performance metrics, suited to standard business spreadsheet or BI software. In contrast, an analyst supporting policy decisions tends to use complex methods that are more suited to being expressed in code, and require more formal methods to ensure quality, such as reproducibility.
+
+In addition, MI is something that nearly every team in the department needs to do, is likely to be closely coupled with the service's systems and business needs. Understanding this environment and user needs across the department is not something that has been the focus of AP. So whilst there is considerable overlap - data storage, processing and dashboards - AP has not been designed for this environment. Indeed there is resistance to adding point-and-click BI software to AP, since the analysts are being encouraged to work in a reproducible way.
+
+### Real time data
+
+AP has Airflow, which can schedule data flows and processing as frequently as every few minutes. However as you move towards on-going data processing like this you start to benefit from data streaming technology, to provide real-time results, cope with disturbances in the flow rate, manage processing in small batches etc. 
+
+Technically there is a clear route to enabling data streaming on AP, such as with Apache Kafka or managed cloud services. However it has not yet been explored yet.
+
+### Data archival
```

Comment:
I like this section to head off future filedump requests. But we need to be careful not to suggest that we don't archive e.g. NOMIS data, since we do.